semantic-conventions/src/tests/semconv/model/test_correct_parse.py

```diff
@@ -479,39 +443,6 @@ def test_extends(self):
         }
         self.semantic_convention_check(list(semconv.models.values())[2], expected)
 
-    def test_include(self):
-        semconv = SemanticConventionSet(debug=False)
-        semconv.parse(self.load_file("yaml/http.yaml"))
-        semconv.parse(self.load_file("yaml/faas.yaml"))
-        semconv.parse(self.load_file("yaml/general.yaml"))
-        semconv.finish()
-        self.assertEqual(len(semconv.models), 10)
-
-        faas_http = [s for s in semconv.models.values() if s.semconv_id == "faas.http"][
-            0
-        ]
-        expected = {
-            "id": "faas.http",
-            "prefix": "faas",
-            "extends": "faas",
-            "n_constraints": 2,
```

Comment:
shouldn't we remove just the constraint check? Or did I miss something?

Reply:
constraints have `include` mode which  adds all the stuff from included semconv into another one (similarly to `extends`).

since I'm removing them, included attributes dissapear from tests where constraints were used.